indico/queries/datasets.py

```diff
@@ -186,12 +186,14 @@ def __init__(
         wait: bool = True,
         from_local_images: bool = False,
         image_filename_col: str = "filename",
+        image_upload_batch_size: int = 20,
```

Comment:
I think this should just be batch size ignore image

Reply:
Updated. Latest commit also has some additional type annotation that was missing on `UploadBatched`

Reply:
nm... I get that you meant not just changing name. It clicked that there are other data types that can be uploaded. Added this and test for pdf.